### ISSUE TITLE
[Yoshino][Lilac] crash during boot with v8 oem

### ISSUE THREAD

**Issue by stefanhh0** [sonyxperiadev/bug_tracker#370]
**Platform:** Yoshino
**Device:** Lilac
**Kernel version:** 4.9.168-g4ebcb68cc743
**Android version:** android-9.0.0_r35
**OEM Image:** SW_binaries_for_Xperia_Android_9.0_2.3.2_v8_yoshino.img
**Build:** aosp_g8441-userdebug 9 PQ2A.190405.003 eng.stefan.20190418.191851
**Version baseband:** 1307-7511_47.1.A.16.20

**Description**
The phone crashed during boot while not being connected to usb.
From the AOSP build following images were flashed:
boot.img, system.img and vendor.img I did not flash the userdata.img.
Also Magisk-v19.0 and open_gapps-arm64-9.0-pico-20190418 have been installed.  

**Symptoms**
The phone was booted fine into aosp, then I triggered a reboot while not being connected to usb. On that boot the phone crashed during the boot phase and rebooted again. A console-ramoops was written.

**How to reproduce**
Occurred until now only once, I don't know yet if it is reproducible.

**Files**
Pstore content: [pstore.tar.gz](https://github.com/sonyxperiadev/bug_tracker/files/3096257/pstore.tar.gz)
Dmesg from boot (after the crash and successful boot) : [dmesg.log](https://github.com/sonyxperiadev/bug_tracker/files/3096259/dmesg.log)

**Comment by stefanhh0** [comment]
Just had a hang and crash when booting while the phone was connected to usb. Phone freezed while showing the black Sony logo on white boot screen. Some thin vertical colorful lines were painted. It remained there for several seconds (~10s) then it rebooted on its own. On that "after crash reboot" the phones vibrator was activated twice (so maybe a double reboot?)

I've saved the pstore content from that occurrence and the dmesg file after the phone booted into the AOSP after that occurrence:
[pstore.tar.gz](https://github.com/sonyxperiadev/bug_tracker/files/3096382/pstore.tar.gz)
[dmesg.log](https://github.com/sonyxperiadev/bug_tracker/files/3096385/dmesg.log)

**Comment by stefanhh0** [comment]
**Platform:** Yoshino
**Device:** Lilac
**Kernel version:** 4.9.168-g72f952b6f2b5
**Android version:** android-9.0.0_r35
**OEM Image:** SW_binaries_for_Xperia_Android_9.0_2.3.2_v8_yoshino.img
**Build:** aosp_g8441-userdebug 9 PQ2A.190405.003 eng.stefan.20190423.201213
**Version baseband:** 1307-7511_47.1.A.16.20

Just an update, also with a more recent AOSP got a crash today while playing audio via BT, I think the vibrator was activated twice, but I am not 100% sure though unfortunately:
[pstore.tar.gz](https://github.com/sonyxperiadev/bug_tracker/files/3114004/pstore.tar.gz)

**Comment by stefanhh0** [comment]
This is old now with all the latest changes... when I am getting another one of these I'll just open a new ticket. Therefore closing this now.
